### DISCUSSION TITLE
Global Settings -> GUI Behaviour tab

### DISCUSSION THREAD

**Discussion by kybos** [muse-sequencer/muse#1129] [Team Posts]
So let's try this new communication channel ;-).

The GUI Behaviour tab would probably deserve some splitting up, there are already too many options and new ones are coming all the time. Perhaps a new tab like GUI tweaks for the more esoteric options, and put the more common options nicely laid out on the GUI Behaviour tab (without the scrolling area?). E.g. the "Track height" seems quite important to me, but it's lost in the long option list).

I think this would be independent of Robert's plans to unite the Global with the Appearance settings.

**Comment by spamatica** [comment]
I agree it is a bit hard to visualize specific settings in the list. If you have ideas on how to organize it I'm all for trying.

Regarding Appearance,saying that I have plans is a bit of a stretch. I seldom have plans, they tend to fall apart ;) I have opinions though :)
